src/cli/subcommands/snapshot_cmd.rs

```diff
@@ -42,6 +49,17 @@ pub enum SnapshotCommands {
         #[arg(long, value_enum, default_value_t = FilecoinSnapshotVersion::V2)]
         format: FilecoinSnapshotVersion,
     },
+    /// Show status of the current export.
+    ExportStatus {
+        /// Wait until it completes and print progress.
+        #[arg(long)]
+        wait: bool,
```

Comment:
Having `--wait` as part of `status` looks a bit weird to me. How about another ExportWait subcommand? (Which could come in a subsequent PR)